mmdet/models/losses/iou_loss.py

```diff
@@ -472,3 +554,66 @@ def forward(self,
             avg_factor=avg_factor,
             **kwargs)
         return loss
+
+
+@LOSSES.register_module()
+class AlphaIoULoss(nn.Module):
+
+    def __init__(self,
+                 eps=1e-9,
+                 reduction='mean',
+                 loss_weight=1.0,
+                 mode='iou',
+                 alpha=3):
+        """AlphaIoULoss.
+
+        Computing the AlphaIoU loss between a set of predicted bboxes
+        and target bboxes.
+
+        Args:
+            eps (float): Eps to avoid log(0). We test serval exps and
+                        1e-9 performs better.
+            reduction (str): Options are "none", "mean" and "sum".
+            loss_weight (float): Weight of loss.
+            mode (str): the calculation method of IoU,
+                        options are "iou" (default), "giou", "diou" and "ciou".
+            alpha (int): adaptively reweighting the loss and gradient of IoU
+        """
```

Comment:
The docstring of __init__ should between `class AlphaIoULoss` and `def __init__`.